ToDo.Core/ViewModels/ToDoViewModel.cs

```diff
@@ -74,7 +75,7 @@ private async Task LoadTodos()
             ToDos = todosCollection;
         }
 
-        private async Task RemoveTodo(Models.ToDo toDoItem)
+        internal async Task RemoveTodo(Models.ToDo toDoItem)
```

Comment:
Just one note: Since the tooling was not soo good back in the days, there is the convention that all async methods end with `Async`
We do not always follow this guideline, but it does make sense to do so.
So you do **not** have to change it now - but please remember it for future features.